src/Console/Command/FixCommand.php

```diff
@@ -105,7 +106,8 @@ protected function configure()
                     new InputOption('diff', '', InputOption::VALUE_NONE, 'Also produce diff for each file.'),
                     new InputOption('format', '', InputOption::VALUE_REQUIRED, 'To output results in other formats.'),
                     new InputOption('stop-on-violation', '', InputOption::VALUE_NONE, 'Stop execution on first violation.'),
-                    new InputOption('show-progress', '', InputOption::VALUE_REQUIRED, 'Type of progress indicator (none, run-in, or estimating).'),
+                    new InputOption('show-progress', '', InputOption::VALUE_REQUIRED, 'Type of progress indicator (none, run-in or estimating).'),
+                    new InputOption('columns', '', InputOption::VALUE_REQUIRED, 'Number of columns to use for progress output (integer or "max").'),
```

Comment:
> -1 for having 2 CLI option for managing progress indicator.
, what is the real usage of later option? 

so we won't have to double the two current outputs;
* run-in
* estimating
* run-in-max
* estimating-max

etc. Secondly the diff and report formatters could also respect the second param later on.

>why would anyone set columns to 25, 87 or 142 ?

dunno, if it is a flag (i.e. boolean for max or non-max i.e. 80 chars. fixed) I'm fine with that as well.